js/activity.js

```diff
@@ -1509,6 +1509,20 @@ define(MYDEFINES, function (compatibility) {
                 [8, 'hidden', 0, 0, [0, null]]
             ];
             blocks.loadNewBlocks(newNote);
+            if (blocks.activeBlock !== null){ // If there exists an active block
```

Comment:
Let's restrict this to note value blocks (any block could be active and thus your assumptions may fail).

Reply:
Is there a simple way to check if there's a dock at the end of a block? That way the user can chain off whatever they were last working on.